src/dmtcp_launch.cpp

```diff
@@ -35,7 +35,7 @@
 using namespace dmtcp;
 
 static void processArgs(int *orig_argc, char ***orig_argv,
-                        string *tmpDir_p, const char **host,
+                        string *tmpDir_p, string &host,
```

Comment:
An earlier commit (df0fe3ff30109f16fa6e33915efab196f065b9c0) had changed the argument type to a string reference. I had missed out on fixing the argument type in the declaration. I noticed this issue when I compiled with `-Wall`.